draft-ietf-quic-recovery.md

```diff
@@ -1526,7 +1530,7 @@ SetLossDetectionTimer():
     loss_detection_timer.cancel()
     return
 
-  // Determine which PN space to arm PTO for.
```

Comment:
stale comment?

Reply:
Let's just remove this comment.

Reply:
Do you want me to be making updates to the PR, or can you take care of anything that's needed?

Reply:
I'm happy to do it.

Reply:
Thank you!